src/utils/job_params_util.py

```diff
@@ -92,6 +92,9 @@ def __init__(self, params, quota, metadata, config):
         self.metadata = metadata
         self.config = config
 
+        # Allow partial node for worker for internal jobs.
+        self.internal = params.get("_internal", False)
```

Comment:
We may need to check the permission in the future.

Reply:
added